bundles/AdminBundle/templates/admin/index/index.html.twig

```diff
@@ -600,12 +600,11 @@
     "pimcore/object/gridexport/xlsx.js",
 
     "pimcore/report/settings.js",
-    "pimcore/report/analytics/settings.js",
-    "pimcore/report/analytics/elementoverview.js",
-    "pimcore/report/analytics/elementexplorer.js",
-    "pimcore/report/webmastertools/settings.js",
-    "pimcore/report/tagmanager/settings.js",
-    "pimcore/report/custom/definitions/analytics.js",
+    "pimcore/report/custom/item.js",
```

Comment:
These files resulting in a 404 error. 
These files do not exist anymore right and should either be loaded by the CustomReportsBundle or the GoogleMarketingBundle itself?